### PR TITLE
feat(LLM/Deeplinks): Adding Deeplinks support for Storyly in LLM

### PR DESCRIPTION
**Pull request by mle-gall**
### 📝 Description

Adding support for Storyly deeplinks in LLM

https://github.com/LedgerHQ/ledger-live/assets/39890664/4c4d8190-8aca-4bbc-a2b4-e073be4e3f1a

### ❓ Context

- **JIRA or GitHub link**:  [LIVE-9794]

### ✅ Checklist

Pull Requests must pass the CI and be code reviewed. Set as Draft if the PR is not ready.

- [x] `npx changeset` was attached.
- [x] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - 

---

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.
- [ ] **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-9794]: https://ledgerhq.atlassian.net/browse/LIVE-9794?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-mobile/src/components/StorylyStories/StorylyProvider.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      const { searchParams } = storylyUrl;
+      const storylyQuery = Object.fromEntries(searchParams);
+      console.log(storylyQuery);
+      if (storylyQuery?.instance)
+        setToken(getTokenForInstanceId(stories as StoriesType, storylyQuery?.instance));
+    }
+  }, [stories, url]);
+
+  const handleLoad = () => {
+    console.log(url);
```
Comment (mcayuelas-ledger) [inline apps/ledger-live-mobile/src/components/StorylyStories/StorylyProvider.tsx:50]: 🗑️

File: apps/ledger-live-mobile/src/components/StorylyStories/StorylyProvider.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    params: { stories },
+  } = useFeature("storyly") || {};
+
+  useEffect(() => {
+    console.log("start", url);
+    if (url) {
+      const storylyUrl = new URL(`ledgerlive://${url}`);
+      const { searchParams } = storylyUrl;
+      const storylyQuery = Object.fromEntries(searchParams);
+      console.log(storylyQuery);
```
Comment (mcayuelas-ledger) [inline apps/ledger-live-mobile/src/components/StorylyStories/StorylyProvider.tsx:43]: 🗑️

**Review by mcayuelas-ledger — COMMENTED** [review]
How does the translations works for the stories ?

**Comment by mle-gall** [comment]
> How does the translations works for the stories ?

The groups have stories in all the supported language directly in Storyly's dashboard.

**Review by ofreyssinet-ledger — APPROVED** [review]
👌

**Review by mcayuelas-ledger — APPROVED** [review]

**Pull request merged**